src/NexusMods.App.UI/LeftMenu/Items/ApplyControlView.axaml

```diff
@@ -41,7 +41,7 @@
                     Height="44"
                     x:Name="ApplyButton">
                 <ToolTip.Tip>
-                    <TextBlock Text="{x:Static resources:Language.ApplyControlView_Apply_ToolTip}" />
+                    <TextBlock Text="{x:Static resources:Language.ApplyControlView_Apply_ToolTip}" Foreground="White" />
```

Comment:
We are using a set of color aliases found in `src/Themes/NexusMods.Themes.NexusFluentDark/Resources/Palette/Colors/ElementColors.axaml` for the colors.
We want to avoid using hardcorded colors like "White" or "Red" to allow us to change them from the theme all throughout the app easily.

In this case the easiest way to get what we want is to just apply the "ForegroundStrong" style class to the `TextBlock`, which will in turn apply `NeutralStrong` as the foreground color.
```suggestion
                    <TextBlock Text="{x:Static resources:Language.ApplyControlView_Apply_ToolTip}" Classes="ForegroundStrong" />
```

Reply:
Hmm, I tried this initially to no avail.

I'll have to look in to why that's happening, it's why I ended up settling for forcing it to white

Reply:
As an alternative you can set it like this:
```suggestion
                    <TextBlock Text="{x:Static resources:Language.ApplyControlView_Apply_ToolTip}" Foreground="{StaticResource NutralStrong}" />
```